ts/components/dialog/debug/components.tsx

```diff
@@ -85,6 +86,63 @@ const CheckVersionButton = ({ channelToCheck }: { channelToCheck: ReleaseChannel
   );
 };
 
+const ForceUpdateButton = () => {
+  const [loading, setLoading] = useState(false);
+  const state = useAsync(async () => {
+    const userEd25519KeyPairBytes = await UserUtils.getUserED25519KeyPairBytes();
+    const userEd25519SecretKey = userEd25519KeyPairBytes?.privKeyBytes;
+    return userEd25519SecretKey;
+  });
+
+  const handleForceUpdate = async () => {
```

Comment:
maybe rename this to something less close to the forceUpdate hook.

Like CheckForUpdateButton and handleCheckForUpdate?

Reply:
Good idea. Fixed